content/en/blog/_posts/2021-03-03-deletes-finaliers-and-owner-references.md

```diff
@@ -0,0 +1,270 @@
+---
+layout: blog
+title: 'Deletes with Finaliers and Owner References'
+date: 2021-03-03
+slug: deletes-finaliers-and-owner-references
+---
+
+**Authors:** Aaron Alpar (Kasten)
+
+Deleting objects in Kubernetes can be challenging. You may think you’ve deleted something, only to find it still persists. While issuing a `kubectl delete` command and hoping for the best might work for day-to-day operations, understanding how Kubernetes `delete` commands operate will help you understand why some objects linger after deletion. 
+
+In this post, we’ll look at:

```

Comment:
In docs (and documentation-like blog articles), [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we):
```suggestion
In this post, I’ll look at:
```